core/src/main/java/com/graphhopper/storage/EdgeAccess.java

```diff
@@ -73,12 +73,14 @@ final void setDist(long edgePointer, double distance) {
      * Translates double distance to integer in order to save it in a DataAccess object
      */
     private int distToInt(double distance) {
```

Comment:
Can you please double check this method ? I was trying to follow the same logic as in CHGraphImpl, but for example I am not really sure why there is -1 in MAX_DIST

Reply:
Likely due to `>=` vs. `>`. Maybe we should somehow use the code from FactorialDecimalEncodedValues so we do not introduce inconsistency and use all 32 bits